.drone.star

```diff
@@ -781,9 +802,10 @@ def localApiTestPipeline(ctx):
                             "steps": skipIfUnchanged(ctx, "acceptance-tests") +
                                      restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin") +
                                      ocisServer(storage, params["accounts_hash_difficulty"], extra_server_environment = params["extraServerEnvironment"]) +
+                                     waitForEmailService(params["emailNeeded"]) +
                                      localApiTests(suite, storage, params["extraEnvironment"]) +
                                      failEarly(ctx, early_fail),
-                            "services": redisForOCStorage(storage),
```

Comment:
is this function used somewhere else? Can we remove this function entirely?

Reply:
Also this says for redis for OC storage, we don't run tests in OC storage anymore

cc @saw-jan

Reply:
didn't see the actual usage. maybe we can remove it.
CC @phil-davis

Reply:
But be done in other PRs if necessary